### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by harshakarampudi**
Please review and let me know if you see any issues. If code looks good please deploy to prod.

### CODE REVIEW

**Comment by awsdevops1999** [comment]
this looks good.

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+who
```
Comment (awsdevops1999) [inline test.sh:1]: can u make changes in the code

Comment (harshakarampudi) [reply test.sh:1]: ok done

**Review by awsdevops1999 — APPROVED** [review]
this pull request looks good

**Pull request merged**